homeassistant/components/light/hue.py

```diff
@@ -47,7 +47,7 @@
 
 PHUE_CONFIG_FILE = 'phue.conf'
 
-SUPPORT_HUE_ON_OFF = (SUPPORT_FLASH | SUPPORT_TRANSITION | SUPPORT_FLASH)
+SUPPORT_HUE_ON_OFF = (SUPPORT_FLASH | SUPPORT_TRANSITION)
```

Comment:
Although for the lightify plug it could make sense to remove the SUPPORT_FLASH altogether. It uses a relais and is rather slow. I'm not sure for how many switch operations the relais is specified.